README.md

```diff
@@ -0,0 +1,27 @@
+[![](http://img.shields.io/travis/{{USER}}/brew.svg?style=flat-square)](https://travis-ci.org/{{USER}}/brew) ![](https://img.shields.io/badge/license-MIT-007EC7.svg?style=flat-square)
+
+<div align="center"> <a href="http://github.com/fish-shell/oh-my-fish"> <img width=90px  src="https://cloud.githubusercontent.com/assets/8317250/8510172/f006f0a4-230f-11e5-98b6-5c2e3c87088f.png"> </a></div><br>
+
+brew
+====
+
+Plugin for [Oh My Fish](https://www.github.com/fish-shell/oh-my-fish) to autocomplet commands of [Homebrew](http://brew.sh).
+
+Install
+-------
+
+```fish
+$ omf install brew
+```
+
+Usage
```

Comment:
You might want to remove `usage`... or just add `brew`... this plugin does not change the usage in any way.